triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/dao/DbTriggerDao.scala

```diff
@@ -127,6 +127,19 @@ class DbTriggerDao(xa: Connection.T) extends RunningTriggerDao {
       }
     } yield (pkgId, payload)
 
+  private def selectRunningTriggers: ConnectionIO[Vector[(UUID, String, String)]] = {
+    val select: Fragment = sql"select * from running_triggers"
```

Comment:
Wildcard selects with no ordering creep me out a bit. That is easy to mess up if something changes.

Reply:
Happy to order by trigger instance (uuid), but I don't know what might get messed up by a change in ordering. We would restart triggers in a different order, which should be fine. I am fine with ordering if that is generally best practice.

Reply:
Might not matter much today but  ime it’s easy to fall into the trap where you assume things are ordered and it works fine in your tests but then explodes later so imho ordering everything is a good idea.

Reply:
Ok, will do then.